templates/metadata_specification.html

```diff
@@ -0,0 +1,53 @@
+{% extends "base/base.html" %}
+{% load markdown %}
+{% load static %}
+{% load format_metadata_field_type %}
+
+{% block content %}
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-one-quarter" id="sticky-sidebar">
+        <ul class="govuk-list">
+            {% for entity, schema_json in entities.items %}
+                <li class="term-group">
+                    <div><a class="govuk-link govuk-link--no-visited-state govuk-link--no-underline" href="#{{ entity }}">
+                        <strong>{{ schema_json.title }}</strong>
+                    </a></div>
+                </li>
+            {%endfor%}
+        </ul>
+    </div>
+    <div class="govuk-grid-column-three-quarters" id="metadata-content">
+        <p class="govuk-body">
+            These classes are used by find-moj-data to represent persisted entities and validate user-generated entities.
+            Classes build upon <a href="https://datahubproject.io/docs/metadata-modeling/metadata-model/#the-core-entities">
+                DataHub's metadata model.
+            </a>
+        </p>
+        {% for entity, schema_json in entities.items %}
+        <table class="govuk-table">
+            <caption class="govuk-table__caption govuk-table__caption--m" id="{{ schema_json.title }}">
+                {{schema_json.title}}
+            </caption>
+            <caption class="govuk-table__caption"><span style="font-weight:normal">{{schema_json.description}}</span></caption>
```

Comment:
Rather than wrapping this is in a span, I think you can add one of these helper classes to the caption: https://design-system.service.gov.uk/styles/font-override-classes/#:~:text=%3C/p%3E-,Font%20weight,-As%20with%20the

It's generally a good idea to avoid inline styles in the html as it makes it hard to keep stuff consistent.